### PR TITLE
Adding Leaflet + CARTO content

### PR DESCRIPTION
**Pull request by ernesmb**
@ramiroaznar  @jsanz

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    	<div id='map'></div>
+    	<script>
+      
+      		var map = L.map('map').setView([51.505, -0.09], 4);
+
+      		L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+        		attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+      		}).addTo(map);
+			
+			L.geoJSON(populated_places, {}).addTo(map);
```
Comment (ernesmb) [inline src/index.html]: this renders slow as hell. I'll need to reduce the dataset

Comment (jsanz) [reply src/index.html]: yea, that's for sure too much for a browser, just render the capitals or something like that: between 100 and 500 points as maximum

File: secciones/leaflet.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+- Getting started
+- Adding a GeoJSON layer
+- Styling a GeoJSON layer
+- Adding a Pop-up
+
+<!-- /MarkdownTOC -->
+
+## Intro to LeafletJS
+
+## Getting started
```
Comment (jsanz) [inline secciones/leaflet.md:18]: You'll comment about adding tiled layers here right?

File: src/index.html (hunk 1 above)
Comment (ernesmb) [reply src/index.html]: yep, too much :_)

I'll be using only spanish cities instead

File: secciones/leaflet.md (hunk 2 above)
Comment (ramiroaznar) [reply secciones/leaflet.md:18]: I think I should translate mine into English as well.

**Pull request merged**